### PR TITLE
Onnx example

### PR DESCRIPTION
**Pull request by jerryzh168**

### CODE REVIEW

File: app/src/main/cpp/native-lib.cpp [hunk 1]
```diff
@@ -173,7 +173,7 @@ Java_facebook_f8demo_ClassifyCamera_classificationFromCaffe2(
     stringStream << avg_fps << " FPS\n";
 
     for (auto j = 0; j < k; ++j) {
-        stringStream << j << ": " << imagenet_classes[max_index[j]] << " - " << max[j] * 100 << "%\n";
+        stringStream << j << ": " << imagenet_classes[max_index[j]] << " - " << max[j] / 10 << "%\n";
```
Comment (bwasti) [inline app/src/main/cpp/native-lib.cpp]: what's going on here? is there no softmax in the last layer of this predict net?

Comment (jerryzh168) [reply app/src/main/cpp/native-lib.cpp]: Yeah, that's correct, last operators are AveragePool and Reshape.

Comment (jerryzh168) [reply app/src/main/cpp/native-lib.cpp]: Since I'm swapping out the pbs as well, I think it should be fine?

**Comment by bwasti** [comment]
yep looks good, I'll merge it :)

**Pull request merged**